test/src/util.ts

```diff
@@ -67,6 +67,54 @@ export function transpileAndExecute(tsStr: string): any {
     return executeLua(transpileString(tsStr));
 }
 
+export function parseTypeScript(typescript: string, target: LuaTarget = LuaTarget.Lua53)
```

Comment:
This looks a lot like transpileString from `Compiler.ts`, maybe we can move the dummy compilerHost creation to a shared function.

Reply:
Maybe we should look at this later. I have tried to figure out a nice way to do this but was not able to come up with a way to neatly refactor this. I don't want to public expose this function from compiler.js, and I'm not sure it would fit at any other location.